src/services/searchSkylink.js

```diff
@@ -80,20 +80,6 @@ export default async function searchSkylink(input) {
     }
   }
 
-  // any arbitrary url
-  if (input.startsWith("https://")) {
```

Comment:
I decided to remove this seeing as we perform a HEAD call at the very bottom of the function, so as a last resort, and ignore the error. So in case it wasn't a valid URL there's no harm done really.